docs/syntax_guide.rst

```diff
@@ -16,6 +16,7 @@ Primitive Data Types
 `Headings <Heading>`   		    representing orientations in space
```

Comment:
Need to add Orientations here and update the entries for Headings and Vector Fields.